src/confluence/subcommands/merge.py

```diff
@@ -26,6 +26,7 @@
 from enum import Enum, auto
 from ..core.setup import setup_logging
 from ..io import read, write
+from .validator import validate
```

Comment:
As a matter of course moving forward, please keep these more atomic, that is, each related functional change goes into its own pull request; e.g. all IO changes (adding JSON, CSV, text, etc.), additional features (such as the validator), etc. These should then be in their own test files; e.g. IO --> test_io.py, validate --> test_validate.py, etc.